### PR TITLE
Revise chewing fuzzer

### PR DESCRIPTION
**Pull request by kcwu**
1. Specify unique name for temp userdb. So we can run multiple fuzzers
   at the same time.
2. Reorganized as three fuzzers for different variants. So we can
   prioritize easily (default first, dynamic config last)

### CODE REVIEW

File: libchewing/Dockerfile [hunk 1]
```diff
@@ -19,4 +19,4 @@ MAINTAINER kcwu@csie.org
 RUN apt-get install -y make autoconf automake libtool texinfo
 
 RUN git clone https://github.com/chewing/libchewing.git
-COPY build.sh chewing_fuzzer.c /src/
+COPY build.sh chewing_fuzzer_common.[ch] chewing_*_fuzzer.c /src/
```
Comment (Dor1s) [inline libchewing/Dockerfile]: I don't see these files. Am I wrong or did you forget to `git add` them?

Comment (kcwu) [inline libchewing/Dockerfile]: I forgot. Now added.

**Comment by Dor1s** [comment]
It fails on Jenkins:

```
[address] Running shell script
+ docker run --user 0 --rm -v /var/lib/jenkins/workspace/libchewing/out/address:/out -t ossfuzz/libfuzzer-runner /out/chewing_dynamic_config_fuzzer -runs=32
INFO: Seed: 394006529
INFO: Loaded 0 modules (0 guards): 
INFO: -max_len is not provided, using 64
INFO: A corpus is not provided, starting from an empty corpus
#0	READ units: 1
Usage: ./chewing_fuzzer [-init] [-extra] [-loop N] [-stdin]
	-init           Random initial configuration
	-extra          Random change all configurations during input.
	                This is usually unexpected.
	-stdin          Get random input from stdin
	-loop N         How many iterations to test (default infinite=-1)

ERROR: script returned exit code 1
Finished: FAILURE
```

**Comment by kcwu** [comment]
oops, it's bug of libchewing.
Fixed by https://github.com/chewing/libchewing/commit/9611ead89ded67f21156fd652aac7443056d497f
It should work now.

**Comment by Dor1s** [comment]
Yes, it works. Thanks!

**Pull request merged**